schema/Core/DomainEmailAddress.entityType.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+return [
+  'name' => 'DomainEmailAddress',
+  'table' => 'civicrm_domain_email_address',
+  'class' => 'CRM_Core_DAO_DomainEmailAddress',
+  'getInfo' => fn() => [
+    'title' => ts('Site Email Address'),
+    'title_plural' => ts('Site Email Addresses'),
+    'description' => ts('Return addresses to use for system-generated emails.'),
+    'log' => TRUE,
+    'add' => '6.0',
+    'icon' => 'fa-envelope',
+    'label_field' => 'display_name',
+    'search_fields' => ['display_name', 'email'],
+  ],
+  'getPaths' => fn() => [],
+  'getIndices' => fn() => [
+    'UI_domain_id_is_default' => [
+      'fields' => [
+        'domain_id' => TRUE,
+        'is_default' => TRUE,
+      ],
+      'add' => '6.0',
+    ],
+  ],
+  'getFields' => fn() => [
+    'id' => [
+      'title' => ts('ID'),
+      'sql_type' => 'int unsigned',
+      'input_type' => 'Number',
+      'required' => TRUE,
+      'description' => ts('Email Address ID'),
+      'add' => '6.0',
+      'primary_key' => TRUE,
+      'auto_increment' => TRUE,
+    ],
+    'name' => [
+      'title' => ts('Display Name'),
+      'sql_type' => 'varchar(63)',
+      'input_type' => 'Text',
+      'required' => TRUE,
+      'description' => ts('Full name of the recipient.'),
```

Comment:
Umm, see my earlier comment about this description. Before I write something else about cheese.